server/api/playbooks.go

```
@@ -609,3 +615,102 @@ func (h *PlaybookHandler) validateMetrics(pb app.Playbook) error {
 	}
 	return nil
 }
+
+func (h *PlaybookHandler) getTopPlaybooksForUser(w http.ResponseWriter, r *http.Request) {
+	userID := r.Header.Get("Mattermost-User-ID")
+	params := r.URL.Query()
+	timeRange := params.Get("time_range")
+	teamID := params.Get("team_id")
+	if teamID != "" && !h.PermissionsCheck(w, h.permissions.PlaybookList(userID, teamID)) {
```

What if `teamID == ""`? What is the expected behavior? Should we return an error?

---

Added checks for teamID = ""